howso/direct/client.py

```diff
@@ -844,6 +842,51 @@ def copy_trainee(
                 f"{new_trainee_name} trainee already exists."
             )
 
+    def copy_subtrainee(
+        self,
+        trainee_id: str,
+        target_trainee: str,
+        *,
+        source_id: Optional[str] = None,
+        source_name_path: Optional[List[str]] = None,
+        target_id: Optional[str] = None,
+        target_name_path: Optional[List[str]] = None,
+    ) -> None:
+        """
+        Copy a subtrainee in trainee's hierarchy.
+
+        Parameters
+        ----------
+        trainee_id : str
+            The id of the trainee whose hierarchy is to be modified.
+        target_trainee: str
```

Comment:
Feels like `target_trainee` should instead be `new_trainee_name` which is what we use for normal copies.

Reply:
Also seems like it should be optional?